app/resources/api/v1/searchable_resource.rb

```diff
@@ -8,18 +8,36 @@ class SearchableResource < JSONAPI::Resource
       key_type :string
 
       def self.find_by_key(text, options = {})
-        search_results = PgSearch.multisearch(text).limit(5).
+        search_results = PgSearch.multisearch(text).
+          where(searchable_type: 'Development').limit(5).
           map(&:searchable).
           map { |record| wrap_with_resource(record) }
-        location = MapzenSearch.new(text).result || null
-        if location.properties['confidence'] > 0.75
-          search_results.unshift LocationResource.new(location, context: {})
-        end
+        loc = MapzenSearch.new(text).result || null
+        search_results.unshift place_resource(text) if no_place?(search_results)
+        search_results.unshift location_resource(loc) if confident_in_location?(loc)
```

Comment:
The unshift logic could be refactored, method's a little big. Maybe this could be its own method.